src/lib/ngsi/Scope.h

```diff
@@ -44,6 +44,8 @@
 #define SCOPE_FILTER_NOT_EXISTENCE   SCOPE_FILTER "::" SCOPE_OPERATOR_NOT "::" "Existence"
 
 #define SCOPE_TYPE_SIMPLE_QUERY      "FIWARE::StringQuery"
+#define SCOPE_TYPE_SIMPLE_QUERY_MD   "FIWARE::StringQuery::Metadata"
```

Comment:
FIWARE::StringQuery::Metadata should be described in documentation (filters.md?). Probably, the same section used for FIWARE::StringQuery could be used as base (copy-paste-modify).